File: 이훈일/삼성 SW 역량 테스트 기출 문제/1234.java
import java.util.Scanner;
import java.io.FileInputStream;
import java.util.*;

class Solution {
	public static void main(String args[]) throws Exception {
		Scanner sc = new Scanner(System.in);
		int T;
		T=10;
		
		for(int test_case = 1; test_case <= T; test_case++) {
			int count = sc.nextInt();
            String secretNum = sc.next();
            String[] secret = secretNum.split("");
            
            Stack<String> stack = new Stack<>();
            for(String num : secret) {
            	if(stack.isEmpty()) {
                	stack.push(num);
                } else {
                	if(stack.peek().equals(num)) {
                		stack.pop();
               		} else {
                    	stack.push(num);
                    }
                }
            }
            
            String answer = "";
            int stackSize = stack.size();
            for(int i = 0; i < stackSize; i++) {
            	answer = stack.pop() + answer;
            }
            
            System.out.println("#" + test_case + " " + answer);
		}
	}
}

File: 이건희/SWexpertacademy/D3/1234.py
for test_case in range(1, 11):
    N, num = map(int, input().split())
    num = list(str(num))
    result = []
     
    for i in num:
        if not result or i != result[-1]:
            result.append(i)
        elif result and i == result[-1]:
            result.pop()
             
    print(f'#{test_case} {int("".join(map(str, result)))}')
